src/main/java/com/base64/d/StudyGuideChecker.java

```diff
@@ -8,127 +8,120 @@
 
 import java.io.IOException;
 import java.util.Base64;
-import java.util.Date;
 import java.util.HashSet;
 import java.util.Set;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class StudyGuideChecker {
-    int count = 0;
```

Comment:
Similar to the comment in `WriteToFile.java`, making these instance variables method variables.